src/sql-parser/src/parser.rs

```diff
@@ -3623,10 +3623,20 @@ impl<'a> Parser<'a> {
         let modifier = self.parse_one_of_keywords(&[SESSION, LOCAL]);
         let mut variable = self.parse_identifier()?;
         let mut normal = self.consume_token(&Token::Eq) || self.parse_keyword(TO);
-        if !normal && variable.as_str().parse() == Ok(TIME) {
-            self.expect_keyword(ZONE)?;
-            variable = Ident::new("timezone");
-            normal = true;
+        if !normal {
+            if variable.as_str().parse() == Ok(TIME) {
+                self.expect_keyword(ZONE)?;
+                variable = Ident::new("timezone");
+                normal = true;
+            }
+            if variable.as_str().parse() == Ok(NAMES) {
+                variable = Ident::new("client_encoding");
+                normal = true;
+            }
+            if variable.as_str().parse() == Ok(SCHEMA) {
+                variable = Ident::new("search_path");
+                normal = true;
+            }
```

Comment:
```suggestion
            match variable.as_str().parse() {
                Ok(TIME) => {
                    self.expect_keyword(ZONE)?;
                    variable = Ident::new("timezone");
                    normal = true;
                }
                Ok(NAMES) => {
                    variable = Ident::new("client_encoding");
                    normal = true;
                }
                Ok(SCHEMA) => {
                    variable = Ident::new("search_path");
                    normal = true;
                }
                _ => (),
            }
```

Reply:
oh I forgot that the Keywords are all the same type so this works!